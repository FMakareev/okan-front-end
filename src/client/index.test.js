import React, {Component} from 'react';
import renderer from 'react-test-renderer';
// import {ROOT} from "./index";
// import { render } from 'react-dom';

// jest.mock('react-dom');

class Link extends Component {
  handleClick = () => {
    alert('Кликнули по ссылке!');
  };

  render() {
    const {title, url} = this.props;

    return <a href={url} onClick={this.handleClick}>{title}</a>;
  }
}

test('ПРоверка', () => {
  const output = renderer.create(
    <Link title="mockTitle" url="mockUrl"/>
  ).toJSON();
  expect(output).toMatchSnapshot();
});
// test('ROOT: Проверка рендера рутового компонента', () => {
//   render(ROOT)
// });
