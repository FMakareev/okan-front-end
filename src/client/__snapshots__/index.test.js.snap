// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ПРоверка 1`] = `
<a
  href="mockUrl"
  onClick={[Function]}
>
  mockTitle
</a>
`;
